addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -324,6 +324,39 @@ export class WysiwygAdapterComponent extends ComponentAdapter {
                     continue;
                 }
 
+                // Skip "animation-play-state" changes only
+                if (record.type === 'attributes' && record.attributeName === 'style') {
+                    const newValue = record.target.style.cssText;
+                    const oldAnimationState = (record.oldValue || '').match(/animation-play-state:\s*([a-z\-]+)/);
+                    const newAnimationState = newValue.match(/animation-play-state:\s*([a-z\-]+)/);
+                    if (oldAnimationState && newAnimationState && oldAnimationState[1] !== newAnimationState[1])
+                        continue;
+                }
+
+                // Skip mutations where only animation-related classes are added/removed
+                if (record.type === 'attributes' && record.attributeName === 'class') {
+                    const oldClasses = (record.oldValue || '').split(/\s+/);
+                    const newClasses = [...record.target.classList];
+
+                    const isAnimClass = cls =>
+                        /^o_anim/.test(cls) || [
```

Comment:
1. I don't like regex-based matching of CSS classes, this has 100% chance of become wrong/dead code in a near future
2. Is not it wrong? Surely if the `o_anim_bounce_in` class is added, I don't want to skip that mutation?